### PR TITLE
UI refactor

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fix : #3327 #3828

### CODE REVIEW

File: public/views/walletHome.html [hunk 1]
```diff
@@ -339,36 +339,35 @@ <h2 class="text-center m10t" translate>My Bitcoin address</h2>
...
-          <div class="small-12 columns" ng-show="index.isCordova && !home.generatingAddress && home.addr[index.walletId]">
-            <span class="button outline light-gray small round expand"
-              ng-click="home.shareAddress(home.addr[index.walletId])">
-              <span translate>Share address</span>
-            </span>
+        </div>
+        <div class="row m10t">
+          <div class="large-12 columns">
+            <div ng-show="home.addr[index.walletId]">
+              <div class="line-t size-10 text-gray m10b p10t" translate>
```
Comment (cmgustavo) [inline public/views/walletHome.html]: Bad position of the `translate` attribute

File: public/views/walletHome.html [hunk 2]
```diff
@@ -339,36 +339,35 @@ <h2 class="text-center m10t" translate>My Bitcoin address</h2>
...
           </div>
-          <div class="small-12 columns" ng-show="index.isCordova && !home.generatingAddress && home.addr[index.walletId]">
-            <span class="button outline light-gray small round expand"
-              ng-click="home.shareAddress(home.addr[index.walletId])">
-              <span translate>Share address</span>
-            </span>
+        </div>
+        <div class="row m10t">
+          <div class="large-12 columns">
+            <div ng-show="home.addr[index.walletId]">
```
Comment (cmgustavo) [inline public/views/walletHome.html]: Move `ng-show` to the `row` *div* and avoid nested *div's*

**Comment by gabrielbazan7** [comment]
Updated

File: public/views/walletHome.html [hunk 3]
```diff
@@ -339,36 +339,31 @@ <h2 class="text-center m10t" translate>My Bitcoin address</h2>
...
-            <span class="button outline light-gray small round expand"
-              ng-click="home.shareAddress(home.addr[index.walletId])">
-              <span translate>Share address</span>
-            </span>
+        </div>
+        <div class="row m10t" ng-show="home.addr[index.walletId]">
+          <div class="large-12 columns">
+            <div class="line-t size-10 text-gray m10b p10t">
+              <span translate> Share this wallet address to receive payments. To protect your privacy, new addresses are generated automatically once you use them.</span>
+              <a ng-click="home.setAddress(true)" ng-disabled="home.blockUx || index.isOffline ||home.generatingAddress" translate>Generate new address</a>
```
Comment (cmgustavo) [inline public/views/walletHome.html]: `ng-disabled` is not available for **a** elements

**Comment by cmgustavo** [comment]
ACK

**Pull request merged**